testing/src/cypress/e2e/04-api-institution/01-instition-dashboard.cy.ts

```diff
@@ -0,0 +1,150 @@
+const COLLEGE_F = Cypress.env("COLL_F");
```

Comment:
IMO the variable name of the test institution can be more generic like TEST_INSTITUTION.

same for USER_F which can be TEST_USER.

Reply:
Renamed it for readability